<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sample16Dialog</class>
 <widget class="QDialog" name="Sample16Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sample16</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelFullName">
       <property name="text">
        <string>Full Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editFullName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelComment">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editComment"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editType"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelADSType">
       <property name="text">
        <string>ADS Type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editADSType"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelSize">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="editSize"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelIndexGroup">
       <property name="text">
        <string>Index Group:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="editIndexGroup"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelIndexOffset">
       <property name="text">
        <string>Index Offset:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="editIndexOffset"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="buttonLoad">
       <property name="text">
        <string>Load Dynamic Symbols</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonParent">
       <property name="text">
        <string>Get Parent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSibling">
       <property name="text">
        <string>Get Sibling</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonChild">
       <property name="text">
        <string>Get Child</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext">
       <property name="text">
        <string>Get Next</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
